### PR TITLE
Link to transcript files

### LINKED ISSUE #93: Link from transcript to transcript on GitHub
At the top of the transcript it should link to that transcript on GitHub so that people can help improve it.

The transcripts are at https://github.com/orbitfm/orbit.fm/tree/master/transcripts

### PR DESCRIPTION
**Pull request by cori**
Fixes #93.

Based on the existing podcast metadata, renaming the files was the most straightforward way to obtain correct urls, as far as I could tell.

### CODE REVIEW

**Comment by cori** [comment]
It occurs to me that by "At the top of the transcript" you may not have meant to link the "Transcript" header itself. If you'd prefer to see a textual link under the header (perhaps something like "Help improve the transcript on GitHub") let me know - i can make that change easily enough.

**Comment by agarrharr** [comment]
Looks pretty good. One problem with this is if the transcript doesn't exist, it should link them to https://github.com/orbitfm/orbit.fm/tree/master/transcripts instead of https://github.com/orbitfm/orbit.fm/tree/master/transcripts where they would get a 404.

File: src/templates/episode.js [hunk 1]
```diff
@@ -222,7 +224,7 @@ export default ({ data }) => {
 
         {transcript && (
           <div>
-            <h1>Transcript</h1>
+            <h1><a href={transcriptLink}>Transcript</a></h1>
```
Comment (agarrharr) [inline src/templates/episode.js]: This should say:
Help improve this transcript on GitHub

File: src/templates/episode.js [hunk 2]
```diff
@@ -143,6 +144,7 @@ const ConnectedTimestamp = connect(
 export default ({ data }) => {
   const episode = data.contentfulEpisode;
   const transcript = data.transcriptsJson && data.transcriptsJson.transcript;
+  const transcriptLink = `https://github.com/orbitfm/orbit.fm/tree/master/transcripts/${episode.podcast.fields.slug}-${leftPad(episode.episodeNumber,3,'0')}.json`;
```
Comment (agarrharr) [inline src/templates/episode.js]: There might also be a season number here:

```
${episode.podcast.fields.slug}-${season? leftPad(episode.season,3,'0'): ''}-${leftPad(episode.episodeNumber,3,'0')}
```

**Comment by cori** [comment]
Thanks for the comments; I believe I address them all.

It seems like it might be worth adding a README.md in the transcripts folder with some guidance about how to help but at the moment I don't think I understand the process well enough to be actually helpful.

File: src/templates/episode.js [hunk 3]
```diff
@@ -251,7 +253,12 @@ export default ({ data }) => {
               ))}
             </div>
           </div>
-        )}
+        ) : (
```
Comment (cori) [inline src/templates/episode.js]: It's probably obvious that I'm a React n00b, but I couldn't figure out how to extract the <h1> out of the conditional so as not to repeat it - I kept getting a jsx compiler error.

Comment (agarrharr) [reply src/templates/episode.js]: Oh ok. I can help you out here. You could do:

```js
          <div>
            <h1>Transcript</h1>
            {transcript ? (
              <h4>
                Help improve this transcript on{' '}
                <a href={transcriptLink}>GitHub</a>
              </h4>
            ) : (
              <h4>
                Help by adding a transcript on{' '}
                <a href={transcriptsLink}>GitHub</a>
              </h4>
            )}
```

**Comment by agarrharr** [comment]
Yep, I'm definitely planning on adding a readme to the transcripts folder.

File: src/templates/episode.js (hunk 3 above)
Comment (cori) [reply src/templates/episode.js]: oh ha - that has nothing to do with being a React n00b. That's just me being a blinkered engineer.

**Comment by cori** [comment]
heh, well `npm run format` really doesn't match the current style, so I rolled that back....

**Pull request merged**